Gemfile.lock

```diff
@@ -268,3 +281,6 @@ DEPENDENCIES
   utf8-cleaner
   will_paginate
   zip-zip
+
+BUNDLED WITH
+   1.10.6
```

Comment:
I really think these gem updates should not happen in this pull request... Adding a feature as contest groups should not involve the risk of updating that many gems. Is it possible to keep the gems the versions they are in master and make this PR work?